AddonCatalog.json

```
@@ -1298,10 +1298,10 @@
   ],
   "yaml-workspace": [
     {
-      "repository": "https://github.com/Mambix/FreeCAD-yaml-workspace",
+      "repository": "https://github.com/Mambix/FreeCAD-yaml-workbench",
       "git_ref": "master",
-      "branch_display_name": "master",
-      "zip_url": "https://github.com/Mambix/FreeCAD-yaml-workspace/archive/refs/heads/master.zip"
+      "branch_display_name": "Master",
```

This is an unrelated change -- let's leave it alone (particularly as it doesn't actually affect anything right now, since the branch display name is only used when there are multiple branches to display).
```suggestion
      "branch_display_name": "master",
```